### PR TITLE
Fix the log sample file

### PR DESCRIPTION
**Pull request by nnDarshan**
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @nthomas-redhat @shtripat PLease review

File: etc/tendrl/node-agent/logging.yaml.timedrotation.sample [hunk 1]
```diff
@@ -2,8 +2,7 @@ version: 1
 disable_existing_loggers: False
 formatters:
     simple:
-        format: "%(asctime)s - %(pathname)s - %(filename)s:%(lineno)s - %
-        (funcName)20s() - %(levelname)s - %(message)s"
+        format: "%(asctime)s - %(filename)s:%(lineno)s - %(funcName)20s() - %(levelname)s - %(message)s"
```
Comment (r0h4n) [inline etc/tendrl/node-agent/logging.yaml.timedrotation.sample]: why did you remove %(pathname)s ?

**Comment by r0h4n** [comment]
Anyways, its fixed
